packages/webapp/components/Proposal/BaseProposals.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  Heading,
+  VStack, Divider, Text, HStack, Stack, Button
+} from "@chakra-ui/react"
+import { FC } from "react"
+import Link from "next/link"
+import { getQuorumVotes, parseState } from "@utils/governanceUtils.ts"
+import { ProposalCard } from "@components/Governance/ProposalCard.tsx"
+import { UserVotes } from "@components/Governance/Delegation/UserVotes.tsx"
+import { DelegateButton } from "@components/Governance/Delegation/DelegateButton.tsx"
+import { isArray } from "lodash"
+import {
+  ProposalState
+} from "@data/contract/requests/getProposalState.ts"
```

Comment:
do these all need the file extension as part of the import?

Reply:
No, but it still works it seems